lib/src/modules/common/constants.dart

```diff
@@ -30,13 +31,13 @@ const kFlutterTagsUrl = 'https://github.com/flutter/flutter/releases/tag/';
 /// Description for the channels
 Map<String, String> channelDescriptions = {
   'stable':
-      '''We recommend that you use this channel for all production app releases. Roughly once a quarter, a branch that has been stabilized on beta will become our next stable branch and we will create a stable release from that branch.''',
+      S.current.weRecommendThatYouUseThisChannelForAllProduction,
```

Comment:
Can you just change these names to stableChannelDescription, where stable is the channle name, this will be a bit shorter and cleaner